src/middleware/errors.middleware.ts

```diff
@@ -1,13 +1,26 @@
 import { NextFunction, Request, Response } from 'express';
 import { HttpException } from '../exceptions';
+import { ClientApiResponse } from '../types';
 
-export const errorMiddleware = (err: HttpException, req: Request, res: Response, next: NextFunction): void => {
-  console.log(err);
-
+export const errorMiddleware = (
+  err: HttpException,
+  req: Request<never, ClientApiResponse<null>, never>,
+  res: Response<ClientApiResponse<null>>,
+  next: NextFunction,
+): void => {
   const status = err.status || 500;
   const message = err.message || 'Something went wrong';
-  res.status(status).json({
+
+  console.log({
     status,
     message,
+    timestamp: new Date().toISOString(),
+    path: req.url,
+  });
```

Comment:
Highly recommended to avoid `console.logs` in PRs code, especially merged to develop 🙂